advocacy_docs/partner_docs/EsriArcGISPro/05-UsingEsriArcGISPro.mdx

```diff
@@ -0,0 +1,484 @@
+---
+title: 'Using Esri ArcGIS Pro with EDB Postgres Advanced Server'
+description: 'Walkthroughs of multiple Esri ArcGIS Pro usage scenarios'
+---
+
+## Create File Geodatabase
+
+Esri has created a new format for storing data with the geodatabase. If you’re creating or saving data, it is recommended to use this format instead of shapefiles. Data stored in a geodatabase will take up less space and perform faster than shapefiles.
+
+1. Go to the HongKong GDB, or another downloaded geodatabase, as instructed in the Download GIS Database steps, and click `Download`.
```

Comment:
For this maybe in the paragraph above we say.  The following steps are going to take your thought creating the geo database using the sample HongKong database.  You can substitute your database in place of this.  
or something like this.

Reply:
"The following steps take you through adding a geodatabase to Esri ArcGIS Pro and creating an Enterprise Geodatabase from that data, using the sample HongKong database.  You can substitute or create your own database in place of this."